### PR TITLE
Upgrade code quality tools

### PR DESCRIPTION
**Pull request by bullder**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,8 +27,8 @@
     },
     "require-dev": {
         "atoum/atoum": "^2.8 || ^3.0",
-        "m6web/coke": "~1.2",
-        "m6web/symfony2-coding-standard": "~1.2",
+        "m6web/coke": "^2.0",
+        "m6web/symfony2-coding-standard": "^3.0",
```
Comment (jdecool) [inline composer.json:31]: Those two packages are now deprecated.

Now we use `php-cs-fixer` to check coding style.
So you should use [php-cs-fixer-config](https://github.com/BedrockStreaming/php-cs-fixer-config) instead.

File: composer.json [hunk 2]
```diff
@@ -27,8 +27,8 @@
     },
     "require-dev": {
         "atoum/atoum": "^2.8 || ^3.0",
```
Comment (jdecool) [inline composer.json:29]: To be compatible with PHP 7.x & 8.x, you shoud upgrade this package.

```suggestion
        "atoum/atoum": "^3.4 || ^4.0",
```

Maybe you will update unit tests

**Review by jdecool — CHANGES_REQUESTED** [review]
Thanks for this PR.

First you should upgrade & replace some dependencies.

After that,  you may fix some breaking changes.

File: composer.json (hunk 1 above)
Comment (bullder) [reply composer.json:31]: I expected that

File: composer.json (hunk 2 above)
Comment (bullder) [reply composer.json:29]: yup, this pr is only about following by requirements from upgraded versions of existed quality tools

**Pull request closed without merging**